expat/configure.ac

```diff
@@ -183,33 +183,53 @@ AC_LINK_IFELSE([AC_LANG_SOURCE([
       AC_MSG_RESULT([yes])],
      [AC_MSG_RESULT([no])])])
 
+AC_ARG_WITH([getrandom],
+  [AS_HELP_STRING([--with-getrandom] [--without-getrandom],
```

Comment:
It would be nice to follow `--with(out)-docbook` help format here:
```
  --with-docbook          enforce XML to man page compilation [default=check]
  --without-docbook       skip XML to man page compilation [default=check]
```
This way it's easer to find and more friendly to humans in general, I would argue.

Reply:
Taken care.